experimental/FTheoryTools/src/AbstractFTheoryModels/attributes.jl

```diff
@@ -1195,33 +1186,222 @@ function chern_class_c2(m::AbstractFTheoryModel; check::Bool = true)
     return get_attribute(m, :chern_class_c2)
```

Comment:
Just to say, this looses any information about the type -- perhaps you can add a type assertion?
```suggestion
    return get_attribute(m, :chern_class_c2)::CohomologyClass
```

Reply:
Thank you. Good pointer. I will adjust this.